### PR TITLE
Add GlusterD 2.0 talk presented in Vault

### PR DESCRIPTION
**Pull request by atinmu**
Signed-off-by: Atin Mukherjee <amukherj@redhat.com>

### CODE REVIEW

File: presentations/index.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+[incontro devops italia](http://www.incontrodevops.it/events/idi2015/) 2016 @ bologna, italy
 --------------------------------------------------------------------------------------------
 
-### April 1, 2016
+### april 1, 2016
 
--   [Gluster roadmap, recent improvements and upcoming features](http://www.incontrodevops.it/sessions/gluster-roadmap-recent-improvements-and-upcoming-features/)
-    ([odp](20160401-Guster-Roadmap.odp), [pdf](20160401-Guster-Roadmap.pdf))
-    Niels de Vos
+-   [gluster roadmap, recent improvements and upcoming features](http://www.incontrodevops.it/sessions/gluster-roadmap-recent-improvements-and-upcoming-features/)
```
Comment (prashanthpai) [inline presentations/index.md]: I don't think these changes are intended. Are they ? If not, please revert these changes.

Comment (atinmu) [inline presentations/index.md]: @prashanthpai : Not sure about why the delta is showing up like this. Original content looks ok, mind to cross check?

Comment (prashanthpai) [inline presentations/index.md]: @atinmu Checked again. The change proposed to add your presentation looks fine. Your commit includes unintended change to other parts. Please verify using `git show` in your local branch.

**Comment by prashanthpai** [comment]
LGTM :+1:  Merging.
